server/src/utils/date_time.rs

```diff
@@ -0,0 +1,15 @@
+//! Utilities to deal with date-time types ( DateTime, Instant, SystemTime, etc)
+
+use std::time::{Instant, SystemTime};
+
+/// Converts an Instant into a SystemTime.
+pub fn instant_to_system_time(instant: Instant) -> SystemTime {
+    SystemTime::now() - instant.elapsed()
+}
+
+pub fn instant_to_unix_timestamp(instant: Instant) -> u64 {
```

Comment:
This method is only used with the processing feature. Please either add `#[cfg(feature = "processing")]` (preferred) or `#[allow(unused)]`. 

Also, can we please add a doc comment?

Reply:
ok, done